csharp/HOLMS.Platform/HOLMS.Platform/Support/ReservationTags/CompStayTag.cs

```diff
@@ -1,9 +1,17 @@
-﻿namespace HOLMS.Platform.Support.ReservationTags {
+﻿using System;
+
+namespace HOLMS.Platform.Support.ReservationTags {
     public class CompStayTag : ReservationTagBase {
         public CompStayTag() { }
 
         public CompStayTag(string[] tokens) { }
 
+        public override bool IsPermanent {
```

Comment:
This makes me a little leery..."permanence" is more of a booking/amendment concern, I'm not sure it's appropriate to expose this to other bits of the system.

I don't absolutely hate it, but let's be careful not to let too much policy "leak out" from the booking context. The intention is that the various contexts can use the tags are pure data, without worrying about sharing too much behavior.